database/raft/Makefile

```diff
@@ -8,21 +8,24 @@ build:
 	createdb core0
 
 run0:
-	LISTEN=127.0.0.1:1999 \
+	LISTEN=localhost:1999 \
 	CHAIN_CORE_HOME=$(HOME)/.chaincore0 \
+	ROOT_CA_CERTS=$(HOME)/.chaincore0/root_ca_certs \
 	DATABASE_URL=postgres:///core0?sslmode=disable \
 	cored
 
 run1:
-	LISTEN=127.0.0.1:1998 \
+	LISTEN=localhost:1998 \
 	CHAIN_CORE_HOME=$(HOME)/.chaincore1 \
-	BOOTURL=http://127.0.0.1:1999/ \
+	ROOT_CA_CERTS=$(HOME)/.chaincore1/root_ca_certs \
+	BOOTURL=https://localhost:1999/ \
```

Comment:
Actually, I would rather remove plain_http from the build tags here if that's okay with you.